zcbor/zcbor.py

```diff
@@ -936,7 +936,7 @@ def multi_member(self):
         return self.multi_var_condition() or self.repeated_multi_var_condition()
 
     def is_unambiguous_value(self):
-        return (self.type == "NIL"
+        return (self.type in ["NIL", "ANY"]
```

Comment:
Should we explain in the commit message why this change is needed?

Reply:
Good point, put it in a separate commit since it wasn't really relevant.